chrome-launcher.ts

```diff
@@ -118,15 +120,17 @@ export class Launcher {
     this.enableExtensions = defaults(this.opts.enableExtensions, false);
     this.connectionPollInterval = defaults(this.opts.connectionPollInterval, 500);
     this.maxConnectionRetries = defaults(this.opts.maxConnectionRetries, 50);
+    this.defaultProfile = defaults(this.opts.defaultProfile, false);
   }
 
   private get flags() {
-    let flags = DEFAULT_FLAGS.concat([
-      `--remote-debugging-port=${this.port}`,
+    let flags = DEFAULT_FLAGS.concat([`--remote-debugging-port=${this.port}`]);
+
+    if (!this.defaultProfile) {
```

Comment:
let's also update the logic for conditionally creating/destroying the tmp directory

Reply:
I'm still using a tmp directory for the logs and the pid file. If I don't use one, should I write those files into the default profile? It can get messy because then I'll have to guess the path of the default profile and delete each file individually.